crates/iceberg-catalog/src/implementations/postgres/tabular/table.rs

```diff
@@ -114,13 +114,16 @@ pub(crate) async fn create_table(
         properties,
     } = request;
 
-    let location = location.ok_or_else(|| {
-        ErrorModel::builder()
-            .code(StatusCode::CONFLICT.into())
-            .message("Table location is required".to_string())
-            .r#type("CreateTableLocationRequired".to_string())
-            .build()
-    })?;
+    let location = location
+        .ok_or_else(|| {
+            ErrorModel::builder()
+                .code(StatusCode::CONFLICT.into())
+                .message("Table location is required".to_string())
+                .r#type("CreateTableLocationRequired".to_string())
+                .build()
+        })?
+        .trim_end_matches('/')
```

Comment:
This shouldn't be part of the Postgres impl, but fixed in catalog.rs so that its fixed for all backends.

Reply:
I'm also adding it to determine_tabular_location, it's also here since most tests do not create tables via the catalog but instead directly on postgres impl level